lib/controllers/ActuatorController.cpp

```diff
@@ -217,4 +217,77 @@ bool ActuatorController::isPastMinAngle() {
 
 bool ActuatorController::isPastMaxAngle() {
   return (getAngle_Degrees() > m_actuatorConfig.maxAngle_Degrees || isLimSwitchMaxTriggered());
+}
+
+// Update m_actuatorConfig upon reception of new PID values
+void ActuatorController::updateActuatorConfigP(const float& value, const bool& velocityPID) {
```

Comment:
Maybe change "value" to be more descriptive such as "new_P_value". From reading the code value doesn't give much info. However the function name is quite explicit in what it is doing so this is just a personal preference.

Reply:
kk will do